### PR TITLE
Wishlist

### PR DESCRIPTION
**Pull request by srijani07**
Wishlist:
image

### CODE REVIEW

File: wishlist.css [hunk 1]
```diff
@@ -0,0 +1,12 @@
+.wishlist-wrapper {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  gap: 1rem;
+}
+
+@media (max-width: 800px) {
+  .wishlist-wrapper {
```
Comment (HarshnaNHaswani) [inline wishlist.css:9]: use ems for media query

File: wishlist.html [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+            <button class="card-product-add-to-cart">Add to cart</button>
+        </div>
+    </div>
+    
+    
+
+                
+
+    
+
```
Comment (HarshnaNHaswani) [inline wishlist.html:96]: Remove extra spacing

**Review by HarshnaNHaswani — APPROVED** [review]
Good to go

File: wishlist.html [hunk 3]
```diff
@@ -0,0 +1,93 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.4.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="https://verve-ui.netlify.app/styles.css" />
+
```
Comment (Bibin-Antony) [inline wishlist.html:9]: reduce unnecessary spaces.

**Review by Bibin-Antony — APPROVED** [review]
seems fine to me. could proceed 👍🏻

**Pull request merged**